client/my-sites/sidebar-unified/menu.jsx

```diff
@@ -96,7 +95,7 @@ export const MySitesSidebarUnifiedMenu = ( {
 					const isSelected = selectedMenuItem?.url === item.url;
 					return (
 						<MySitesSidebarUnifiedItem
-							key={ item.slug }
+							key={ item.title }
```

Comment:
The *Background* and *Customize* items have the same slug. Title should be unique.